src/taco-cli/resources/en/resources.json

```diff
@@ -170,6 +170,8 @@
     "_CommandKitSelectStatusUpdatingPlatforms.comment": "Status message on updating platforms",
     "CommandKitSelectStatusUpdatingPlugins": "Updating plugins...",
     "_CommandKitSelectStatusUpdatingPlugins.comment": "Status message on updating plugins",
+    "CommandKitSelectStatusUpdatingTacoJson": "Updating taco.json...",
```

Comment:
Soak and I prefer just "updating taco.json". More in line with "updating config.xml". what do you think?